BUILDING.md

```diff
@@ -228,28 +227,78 @@ testing. Example:
 TAG=main docker-compose up
 ```
 
+## Nomad
+Grapl is transitioning to Nomad as our container orchestration. This will replace both AWS Fargate and docker-compose.
+
+### Nomad local development setup
+Have docker and docker-compose installed. Install Nomad and Consult following the instructions at 
+https://www.nomadproject.io/downloads and https://www.consul.io/downloads respectively.
+
+Install CNI plugins:
+
+```bash
+sudo mkdir -p /opt/cni/bin
+cd /opt/cni/bin
+sudo wget https://github.com/containernetworking/plugins/releases/download/v1.0.0/cni-plugins-linux-amd64-v1.0.0.tgz
+sudo tar -xf cni-plugins-linux-amd64-v1.0.0.tgz
+```
+
+If you are on ChromeOS you will also need to run the following due to https://github.com/hashicorp/nomad/issues/10902.
+This will allow Nomad to run bridge networks
+```bash
+sudo mkdir -p /lib/modules/$(uname -r)/
+echo '_/bridge.ko' | sudo tee -a /lib/modules/$(uname -r)/modules.builtin
+```
+
+### Running Nomad locally
+
+1. Set up local registry, build and push containers to local registry
+```bash
+make build-docker-images-local
+```
+
+2. Start local Nomad environment
+```bash
+sudo ./nomad/local/start_development_environment.sh
+```
+
+3. Deploy Grapl local-stack dependencies via Nomad
+```bash
+nomad job run nomad/local/grapl-local-infra.nomad
+```
```

Comment:
Looking at `start_development_environment.sh` this looks like it's not a required separate step - it's part of that

Reply:
You're right, removing now